doc/js/rewrite-links.js

```diff
@@ -11,5 +11,7 @@ document.querySelectorAll("a").forEach(function (a) {
       a.target = "_blank";
       a.rel = "noreferrer noopener";
     }
-  } catch {}
+  } catch {
+    /* eslint-disable-line no-empty */
```

Comment:
We could also use this option:  https://eslint.org/docs/latest/rules/no-empty#allowemptycatch